site/src/first-entry.md

```diff
@@ -34,6 +34,12 @@ You may also use the Additional Notes space to add any other information not cov
 
 Once you are done recording your Environmental Sample's information, click "Save" toward the top right of the form.
 
+Make sure that you are deleting accidental entries.
+
+Every time that you add or duplicate an entry, you are creating a new Status:Incomplete even if you immediately hit return on your browser. 
+
+To declutter your online workspace, click "edit" on any entry you want gone and then click "move to trash." 
+
```

Comment:
I don't like this placement for this note. Seems odd to talk about accidental entries and duplicating while we still haven't created our first entry yet.

Currently, we talk about accidental entries first here: https://github.com/Tiny-Earth/DiscoveryDatabaseWorkshop/blob/fc6706b3717efdf899017cf4d79a68d64e1e65a6/site/src/status.md?plain=1#L3

I think a good strategy for this note would be:

1. At the end of [this line](https://github.com/Tiny-Earth/DiscoveryDatabaseWorkshop/blob/fc6706b3717efdf899017cf4d79a68d64e1e65a6/site/src/status.md?plain=1#L3) in `site/src/status.md`, say that we'll show a trick on the next page of the workshop for finding all your Incomplete entries at once
2. In `site/src/my-data.md`, add a new section (`##`) above Views, called Filters, and describe in general terms some of the things you can filter by in the default view of the My Data page
3. In that Filters section, add a `tip` admonition that describes in snappy terms how to use the filters to find all `Status: Incomplete` entries, so that you can delete them